src/dev/flang/util/Map2Int.java

```diff
@@ -136,7 +136,7 @@ public int get(T t)
   /**
    * get the index of an element
    *
-   * @return t's index or _base-1 if t was not added.
+   * @return the T that belongs to index i.
```

Comment:
suggesting to be more explicit here:
```suggestion
   * @param i an index returned by a call to add
   *
   * @return the T passed to add when it returned i.
```